kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -146,11 +146,24 @@ var StarView = BaseView.extend({
     template: require("./hbtemplates/star.handlebars"),
 
     events: {
-        "click .star-rating-option": "rate_value_callback"
+        "click .star-rating-option": "rate_value_callback",
+        "click .star-rating-option >": "rate_value_callback",
+        "mouseenter .star-rating-option >": "mouse_enter_callback",
+        "mouseenter .star-rating-option": "mouse_enter_callback",
+        "mouseleave .star-rating-option": "mouse_leave_callback"
+    },
+
+    label_values: {
+        1: "Very Low",
+        2: "Low",
+        3: "Normal",
+        4: "High",
+        5: "Very High"

```

Comment:
Star rating labels are set here. This could easily be configurable on a per-view basis.